grails-app/assets/javascripts/projects.js

```diff
@@ -240,6 +247,22 @@ function ProjectViewModel(project) {
             });
         }
     });
+
+    /**
+     * Automatically add states of excluded electorates to excluded states field provided that state is
+     * available for selection.
+     */
+    self.geographicInfo.otherExcludedElectorates.subscribe(function(newOtherExcludedElectorates) {
+        var statesOfNewOtherExcludedElectorates = findStatesElectoratesBelong(newOtherExcludedElectorates),
+            availableStates = self.transients.states.statesToExclude();
+
+        statesOfNewOtherExcludedElectorates && statesOfNewOtherExcludedElectorates.forEach(function(state) {
+            if (availableStates.indexOf(state) > -1) {
+                self.geographicInfo.otherExcludedStates.push(state);
```

Comment:
We should only exclude the State associated with the excluded Electorate if there are no other (non excluded) Electorates in that same State - otherwise we are excluding a valid state.